lib/menu.ts

```diff
@@ -108,6 +108,9 @@ export class Menu extends Disposable implements IPopupContent {
   constructor(private ctl: IOpenController, items: DomElementArg[], options: IMenuOptions = {}) {
     super();
 
+    // Indicates if the trigger element is an input.
+    const isInputTrigger = this.ctl.getTriggerElem().nodeName == "INPUT";
```

Comment:
I think what's happening here is you are saying `Menu` when used on an `INPUT` is not a menu but a select-like dropdown. What do you think of creating a separate class (perhaps in a new file) `class Select extends Menu`. Its constructor would call `super()`, then add a few new things to `this.content` and to `triggerElem`. Menu might need to expose its next/prev methods (perhaps protected is enough).

I think Select would only add behavior to Menu, so making it a subclass would nicely separate out the extra logic. The one exception is setTimeout(focus).... I actually think the *best* way to deal with it is to rename `Menu` to `BaseMenu`, have Menu's `constructor` call `super(); setTimeout( menu-specific-logic )`, and have Select's inherit from `BaseMenu`, and have its constructor call `super(); select-specific-logic; no setTimeout`.